doc/arkode/guide/source/Usage/User_callable.rst

```diff
@@ -1824,6 +1825,49 @@ Specify if the implicit RHS is deduced after a nonlinear solve  :c:func:`ARKodeS
    .. versionadded:: x.y.z
 
 
+.. c:function:: int ARKodeSetAutonomous(void* arkode_mem, sunbooleantype autonomous)
+
+   Specifies that the implicit portion of the problem is autonomous i.e., does
+   not explicitly depend on time.
+
+   When using an implicit or ImEx method with the trivial predictor, this option
+   enables reusing the implicit right-hand side evaluation at the predicted
+   state across stage solves within a step. This reuse reduces the total number
+   of implicit RHS function evaluations.
+
+   :param arkode_mem: pointer to the ARKODE memory block.
+   :param autonomous: flag denoting if the implicit RHS function,
+                      :math:`f^I(t,y)`, is autonomous (``SUNTRUE``) or
+                      non-autonomous (``SUNFALSE``).
+
+   :retval ARK_SUCCESS: the function exited successfully.
+   :retval ARK_MEM_NULL: ``arkode_mem`` was ``NULL``.
+   :retval ARK_ILL_INPUT: an argument had an illegal value.
+   :retval ARK_STEPPER_UNSUPPORTED: implicit solvers are not supported by the
+                                    current time-stepping module.
+
+   .. warning::
+
+      Results may differ when combining :c:func:`ARKodeSetAutonomous` with
+      :c:func:`ARKodeSetDeduceImplicitRhs` and using a stiffly accurate method
+      with the trivial predictor as the deduced implicit right-hand side (RHS)
+      value will be reused in the initial residual computation rather than
+      evaluating the implicit RHS function e.g.,
+      ``examples/arkode/C_serial/ark_brusselator.c``.
```

Comment:
I'm not sure that I understand why a code example is used to support this statement.  Perhaps some explanation here would help, such as
```suggestion
      evaluating the implicit RHS function.  This issue can be seen in the example problem 
      ``examples/arkode/C_serial/ark_brusselator.c``, where (explain what happens).
```

Reply:
See [this comment below](https://github.com/LLNL/sundials/pull/495#discussion_r1613540758). I'll update this text with more details.

Reply:
I've updated this text (and added a comment in the example code) to clarify how this example be used to demonstrate the differences that can arise when combining `SetAutonomous` and `SetDedeuceImplicitRhs` with a stiffly accurate method.